scripts/imgtool/main.py

```diff
@@ -450,7 +464,7 @@ def sign(key, public_key_format, align, version, pad_sig, header_size,
          dependencies, load_addr, hex_addr, erased_val, save_enctlv,
          security_counter, boot_record, custom_tlv, rom_fixed, max_align,
          clear, fix_sig, fix_sig_pubkey, sig_out, user_sha, is_pure,
-         vector_to_sign, non_bootable):
+         vector_to_sign, non_bootable, psa_key_ids):
```

Comment:
It should be passed to the Image constructor in line 473.
Currently it's only initialized to None which is then used by the `create` method.

Reply:
I have updated so that psa_key_ids is applied to the image using the set function in the new PS.